common/util.go

```diff
@@ -295,7 +295,7 @@ func RemoveRoleDS(ms *dstate.MemberState, role int64) error {
 }
 
 var StringPerms = map[int64]string{
-	discordgo.PermissionReadMessages:       "Read Messages",
+	discordgo.PermissionViewChannel:        "View Channel",
```

Comment:
I didn't bother keeping the old names for backward compatibility because `StringPerms` is only used in the web UI and there's nothing bad that can happen if that changes a little.